### ISSUE TITLE
Agk2Exe Warning: Autohotkey could not be located

### ISSUE THREAD

**Issue by TheDelta** [imaginationac/compile-ahk#6]
When I try to compile an ahk script, I get the following warning all the time:

`Warning: AutoHotkey could not be located !`
![Ahk2Exe_8WeD8fAbnX](https://user-images.githubusercontent.com/2158550/181901513-3a7ca727-dbb6-409f-865a-69f987f0f9f7.png)

This seems to be related to https://www.autohotkey.com/boards/viewtopic.php?t=78361 and only happens when I check `Set Version Info`.

OS: Windows 10, 64bit
Latest AHK: `1.1.34.03`

**Comment by BeyondVertical** [comment]
Yeah, had that too. I then switched back to the "normal" way of compiling. But I did prefer this solution.
